modules/osdk-scorecard-extending-plugins.adoc

```diff
@@ -0,0 +1,376 @@
+// Module included in the following assemblies:
```

Comment:
I'd probably just remove this file as external plugin support was removed from this version of scorecard.